### DISCUSSION TITLE
RTMP to WebRTC playback URL

### DISCUSSION THREAD

**Discussion by bigbean17** [ant-media/Ant-Media-Server#6272] [Q&A] [answered]
Hi,

I have a camera which publishes video via RTMP and I would like to make my client to view it via WebRTC. I can view the video on the provided web interface but I couldn't view it using webrtc playback url.I built the url using the example in the demo web but still shows "No compatible source was found for this media."
<img width="1046" alt="image" src="https://github.com/ant-media/Ant-Media-Server/assets/69971426/319e5c44-5c0a-4d46-8d35-86d597f918cd">

Any help would be appreciated.

Thank you very much!

**Comment by Mohit-3196** [comment]
Hi @bigbean17,
How are you?

Are you able to play the WebRTC stream with the sample webrtc player page, player.html, or the embedded player, play.html?
https://antmedia.io/docs/guides/playing-live-stream/webrtc-playback/

Thank you.

**Reply by bigbean17** [reply]
Hi @Mohit-3196,

Thanks for the quick reply, I could run it in the embedded player using the code provided on the server web interface. And also available at http://<server>:5080/test/play.html

**Reply by bigbean17** [reply]
The webrtc url I have is ws://<ip_address>:5080/test/<stream_id>.webrtc

**Reply by Mohit-3196** [reply]
Thank you @bigbean17,
Then it seems like there is some issue with the sample created on your end.
Please check this codepen sample to create a webrtc player page https://antmedia.io/docs/guides/developing-antmedia-server/webrtc-play-page-creation-tutorial/

Thank you.

**Reply by bigbean17** [reply]
Still not working, is "ws" an issue? Do I have to use secure websocket (wss) for this? Thank you~

**Reply by bigbean17** [reply]
I think it is the issue of insecure websocket, will install ssl to see how it goes, thanks for helping!

**Reply by Mohit-3196** [reply, marked as answer]
Hi @bigbean17,
It can be about WSS depending on the security policy.
Let me know after you enable SSL.

**Reply by yashtandon113** [reply]
@bigbean17 
After you install the SSL certificate, use the below websocket URL format, and it should work fine.

`wss://domain:5443/LiveApp/streamId.webrtc`

**Reply by bigbean17** [reply]
Hi @Mohit-3196  @yashtandon113, the video worked after we made our site https and use the secure websocket, thanks for helping!

**Reply by Mohit-3196** [reply]
That's great.
Thank you :)
